src/Compiler.cpp

```diff
@@ -344,16 +461,16 @@ TfLiteTensor* )"
   return &g_ctx.tensors[outTensorIndices[index]];
 }
 
-void )"
+TfLiteStatus )"
       << prefix_ << R"(invoke() {
-  TfLiteStatus status = kTfLiteOk;
-)";
-  for (size_t i = 0; i < nodes_.size(); i++) {
-    wr << "  status = g_registrations[" << nodes_[i].regIndex
-       << "]->invoke(&g_ctx, &g_nodes[" << i << "]);\n";
-    wr << "  assert(status == kTfLiteOk && \"Invoke failed\");\n";
+  for(size_t i = 0; i < )" << nodes_.size() << R"(; ++i) {
+    TfLiteStatus status = g_registrations[nodes[i].used_op_index]->invoke(&g_ctx, &g_nodes[i]);
```

Comment:
We only use used_op_index for indexing into g_registrations. Should we directly store a pointer to the g_reg element in the first place?

Reply:
It _feels_ like this _might_ save some bytes (if we find other elements which we can reduce to smaller than integer).